internal/cmd/operator_list_custom_resources.go

```diff
@@ -0,0 +1,76 @@
+package cmd
```

Comment:
Maybe just remove this file for now since it is not critical for feature freeze?

I have opinions, but I don't want to block this PR getting in :)

Reply:
ok

Reply:
will revisit these review comments in a follow-up PR when we add the cmd back :)